plugins/html-plugin/src/main/java/ru/sbtqa/tag/stepdefs/HtmlSteps.java

```diff
@@ -2,12 +2,15 @@
 
 public class HtmlSteps extends HtmlGenericSteps<HtmlSteps> {
 
-    private static HtmlSteps instance;
+    static final ThreadLocal<HtmlSteps> storage = new ThreadLocal<HtmlSteps>() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this anonymous class with a call to "ThreadLocal.withInitial". ([squid:S4065](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS4065))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOCccrQ7oSxgxnf7&open=AWeIHOCccrQ7oSxgxnf7)

[ISSUE_KEY]: AWeIHOCccrQ7oSxgxnf7